### PR TITLE
feat: [CHAOS-2009]: Fixed formix text input type

### PR DESCRIPTION
**Pull request by amityt**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by vanshBhatia-A4k9 — DISMISSED** [review]
LGTM

**Review by arkajyotiMukherjee — DISMISSED** [review]

File: src/modules/73-discovery/pages/home/views/create-discovery-agent/CreateDAgent.tsx [hunk 1]
```diff
@@ -173,9 +173,8 @@ const CreateDAgent: React.FC<DrawerProps> = /* istanbul ignore next */ ({ setDra
               connectorRef: Yup.string().trim().required(getString('discovery.dAgentValidation.connectConnector')),
               discoveryNamespace: Yup.string().trim().required(getString('discovery.dAgentValidation.selectNamespace')),
               duration: Yup.number()
-                .min(0, getString('discovery.dAgentValidation.durationMaxMin'))
-                .max(60, getString('discovery.dAgentValidation.durationMaxMin'))
-                .required(getString('discovery.dAgentValidation.durationRequired')),
+                .min(1, getString('discovery.dAgentValidation.durationMaxMin'))
```
Comment (pankajsingh-harness) [inline src/modules/73-discovery/pages/home/views/create-discovery-agent/CreateDAgent.tsx:176]: Use `getDurationValidationSchema` from `src/modules/10-common/components/MultiTypeDuration/helper.ts` for this validation

Comment (amityt) [reply src/modules/73-discovery/pages/home/views/create-discovery-agent/CreateDAgent.tsx:176]: Hi @pankajsingh-harness , I guess this validation is strictly used for duration with units (1s, 1m etc). In our use case, the backend is accepting the duration in number. Not sure if we can add this validation here.

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by S-ayanide — APPROVED** [review]
LGTM

File: src/modules/73-discovery/pages/home/views/create-discovery-agent/CreateDAgent.tsx (hunk 1 above)
Comment (arkajyotiMukherjee) [reply src/modules/73-discovery/pages/home/views/create-discovery-agent/CreateDAgent.tsx:176]: Yes, This validation doesn't seem applicable in this situation.

**Pull request merged**